installer/helpers/destroy/delete_event_source_mapping.py

```diff
@@ -17,7 +17,7 @@ def delete_event_source_mapping(stackName):
         print("Listing Event source mapping for function " + event_source_function)
         list_event_output = subprocess.check_output([
             "aws", "lambda", "list-event-source-mappings", "--function-name",
-            event_source_function
+            event_source_function, "--region", region
```

Comment:
Can I ask why we're invoking `awscli` via a subprocess when we could just `import boto3` and use the AWS python library to do it?

Reply:
yeah, we can rewrite the delete cloudfront and event source mapping using boto3